apps/supernova/src/App.jsx

```diff
@@ -62,13 +59,31 @@ function App(props = {}) {
   )
 }
 
+const AppWithOldUrlStructure = (props) => {
+  // syncs navigation relevant states with the url for deep links
+  // gets the state from the URL in the beginning
+  // sets the URL from state information
+  useUrlState()
```

Comment:
As discussed, we need to improve how we handle updating the URL state using a hook. Would you like to file an issue so we can use it as a reference for discussions and research?

Reply:
Yes I've added a new refactor task https://github.com/cloudoperators/juno/issues/746